test/html_test.js

```diff
@@ -25,37 +26,70 @@ function run(test, config, expected, message) {
   });
 }
 
-exports.htmllint = {
-  'all': function(test) {
-    run(test, {
-      files: ['test/valid.html', 'test/invalid.html']
-    }, [
-      {
-        lastLine: 1,
-        lastColumn: 16,
-        message: 'Start tag seen without seeing a doctype first. Expected “<!DOCTYPE html>”.',
-        file: path.join('test', 'invalid.html')
-      },
-      {
-        lastLine: 9,
-        lastColumn: 96,
-        message: 'Attribute “unknownattr” not allowed on element “img” at this point.',
-        file: path.join('test', 'invalid.html')
-      },
-      {
-        lastLine: 9,
-        lastColumn: 96,
-        message: 'An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.',
-        file: path.join('test', 'invalid.html')
-      },
-      {
-        lastLine: 11,
-        lastColumn: 19,
-        message: 'The “clear” attribute on the “br” element is obsolete. Use CSS instead.',
-        file: path.join('test', 'invalid.html')
-      }
-    ], 'four errors from test/invalid.html');
-  },
+exports.htmllint = testCase({
```

Comment:
What makes the `testCase` calls required? Seemed to work fine without them before.